src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MAGNIFICATION_FACTOR = 2;
+    private int size;
+    private int threshold = (int) (DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR);
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
to get the value we should do some calculations - let's better do it in the constructor
same for the `table` array - if we will create a constructor to initialize threshold there - we should better move the array creation there as well - it will look better and more readable